extension/lib/doc.js

```diff
@@ -285,23 +375,24 @@ class Doc {
     Sets this.keywords
     Determines the keywords of the document (using simple term frequency filter)
     */
-   setKeyWords(termFreq, documentFreq, num_documents) { 
-    let keywords = new Set();
-    for (var word in termFreq) {
-        let relTermFreqValue = termFreq[word] / this.total_words; // Get relative term frequency
-        let documentFreqValue = documentFreq[word]
-        if (!documentFreqValue) {
-            keywords.add(word.toLowerCase()); // Word not encountered before -> Add to keywords
-            continue;
-        }
-        let relDocumentFreqValue = documentFreqValue / num_documents // Get relative document frequency
-        let tfIdfScore = relTermFreqValue / relDocumentFreqValue; // Score = TF/DF (hence TF-IDF)
-        if (tfIdfScore > keywordFilter) { // Filter by score
-            keywords.add(word.toLowerCase());
+   setKeyWords(termFreq, documentFreq, num_documents, cb) { 
```

Comment:
why have cb as parameter is this code is not async?
- remove cb() as param
- then caller just do setKeywords(); cb();

Reply:
Vestigial bug

Thanks!